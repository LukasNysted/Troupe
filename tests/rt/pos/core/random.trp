(* 

Generate a pair of random numbers and check a
bunch of expected facts about them: 

- each is between between 0 (inclusive) and 1
- (it is unlikely) they are not the same 

*)

let val r1 = random () 
    val r2 = random () 
in (r1 >= 0, r2 >= 0, r1 < 1, r2 <1, r1 <> r2 )  
end    
