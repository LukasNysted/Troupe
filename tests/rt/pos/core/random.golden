2020-03-03T23:40:52.306Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: (true@{}%{}, true@{}%{}, true@{}%{}, true@{}%{}, true@{}%{})@{}%{}
